src/dfx/src/actors/replica.rs

```diff
@@ -309,7 +309,7 @@ fn replica_start_thread(
             cmd.args(&["--http-port", &port.to_string()]);
         }
         if config.btc_adapter.enabled {
-            cmd.args(&["--subnet-features", "bitcoin_testnet"]);
+            cmd.args(&["--subnet-features", "bitcoin_regtest"]);
             if let Some(socket_path) = config.btc_adapter.socket_path {
                 cmd.args(&[
                     "--bitcoin-testnet-uds-path",
```

Comment:
Also, note that the second parameter involved is `--bitcoin-testnet-uds-path`.  Should this be `--bitcoin-regtest-uds-path` (which does not currently exist as a parameter for ic-starter) ?

Reply:
@ericswanson-dfinity This is correct as-is. In theory, this uds-path is not specific to any particular network and should just be `--uds-path`. The networking team is aware of the inaccuracy of the current nomenclature and I expect they'll sort that out at some point. cc @rumenov